addon/components/frost-link.js

```diff
@@ -35,6 +38,17 @@ function addSizeClass (size, classes) {
   }
 }
 
+function addDesignClass (design, classes) {
+  switch (design) {
+    case 'info-bar-action':
+      classes.push('action')
```

Comment:
Also, this isn't really an action is it?  I'd move to 'info-bar' for the design type